src/Components/LandingPage/LandingPage.js

```diff
@@ -38,6 +40,17 @@ https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/up
                                 ">
                                     Documentation
                             </Button>
+                            <br />
+                            <Button
+                                component="a"
+                                target="_blank"
+                                variant="link"
+                                icon={ <GithubIcon /> }
+                                iconPosition="right"
+                                isInline
+                                href={ 'https://github.com/RedHatInsights/image-builder-frontend/commit/' + COMMITHASH }>
```

Comment:
Maybe make this `https://github.com/RedHatInsights/image-builder-frontend/tree/` instead? That way it shows the code at the given commit, rather than the commit itself?